Sources/SwiftProtobuf/JSONEncoder.swift

```diff
@@ -70,7 +70,11 @@ internal struct JSONEncoder {
 
     internal init() {}
 
+    #if swift(>=5.0)
```

Comment:
As above, if the new syntax works in Swift 4, then we can avoid this conditional.

Reply:
I just tried it; the new syntax does seem to work in Swift 4.